### PR TITLE
feat: controller notification admin

### PR DESCRIPTION
**Pull request by AndyTWF**
Allow ops to create notifications for the plugin.

Builds on #1006 

Closes https://github.com/VATSIM-UK/uk-controller-api/pull/840

### CODE REVIEW

**Comment by AndyTWF** [comment]
@AxonC Main merged :)

File: app/Models/Notification/Notification.php [hunk 1]
```diff
@@ -5,20 +5,27 @@
...
 use App\Models\User\User;
 use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\BelongsToMany;
 use Illuminate\Database\Eloquent\SoftDeletes;
 
 class Notification extends Model
 {
+    use HasFactory;
```
Comment (AxonC) [inline app/Models/Notification/Notification.php:15]:
```suggestion
    use HasFactory, SoftDeletes;
```

**Review by AxonC — APPROVED** [review]

File: app/Models/Notification/Notification.php (hunk 1 above)
Comment (AndyTWF) [reply app/Models/Notification/Notification.php:15]: Iirc PSR12 doesn’t like multiple traits per use! 


https://www.php-fig.org/psr/psr-12/#42-using-traits

Comment (AxonC) [reply app/Models/Notification/Notification.php:15]: The PHP docs aren't PSR12 compliant then :D

Comment (AndyTWF) [reply app/Models/Notification/Notification.php:15]: They never are :D

**Pull request merged**